Tasks/Common/webdeployment-common/utility.ts

```diff
@@ -52,7 +52,12 @@ export function getSetParamFilePath(setParametersFile: string) : string {
     else if (!fileExists(setParametersFile)) {
         throw Error(tl.loc('SetParamFilenotfound0', setParametersFile));
     }
-
+    else if(fileExists(setParametersFile)) {
```

Comment:
The responsibility of the function is more than get. It does copy. Can you change the name of the function accordingly.